ipfs-cluster-ctl/formatters.go

```diff
@@ -147,7 +147,7 @@ func textFormatPrintGPInfo(obj *api.GlobalPinInfoSerial) {
 
 	for _, k := range peers {
 		v := obj.PeerMap[k]
-		fmt.Printf("    > Peer %s : %s", k, strings.ToUpper(v.Status))
+		fmt.Printf("    > Peer %-10s : %s", v.PeerName, strings.ToUpper(v.Status))
```

Comment:
If `v.PeerName` is set it should show the PeerName, otherwise it should show the peer ID. Depends on how comment above is resolved.

We can remove the `Peer ` part. I think it's understood and saves some space. We can increase padding to 15. I'd like to see an example of the output though.